roles/kubernetes-apps/metallb/templates/metallb-config.yml.j2

```diff
@@ -12,6 +12,13 @@ data:
     - peer-address: {{ peer.peer_address }}
       peer-asn: {{ peer.peer_asn }}
       my-asn: {{ peer.my_asn }}
+{% if peer.source_address is defined %}
+      source-address: {{ peer.source_address }}
+{% endif %}
+{% if peer.node_selectors is defined %}
+      node-selectors:
+        {{ peer.node_selectors | to_yaml(indent=2, width=1337) | indent(8) }}
```

Comment:
According to https://metallb.universe.tf/configuration/#configuring-the-bgp-source-address the above change seems good for me.